src/features/user/Account/ReactivateModal.tsx

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
-import Modal from '@material-ui/core/Modal';
-import Backdrop from '@material-ui/core/Backdrop';
-import Fade from '@material-ui/core/Fade';
 import { ThemeContext } from '../../../theme/themeContext';
 import styles from './AccountHistory.module.scss';
 import { useTranslation } from 'react-i18next';
 import { putAuthenticatedRequest } from '../../../utils/apiRequests/api';
 import { UserPropsContext } from '../../common/Layout/UserPropsContext';
 import Close from '../../../../public/assets/images/icons/headerIcons/close';
 import { ErrorHandlingContext } from '../../common/Layout/ErrorHandlingContext';
-import { CircularProgress } from '@material-ui/core';
+import { CircularProgress, Modal, Fade } from '@mui/material';
```

Comment:
There are slight differences in font styles and sizes, spacing, height and width of modal. 

![Screenshot 2022-05-05 at 5 36 36 PM](https://user-images.githubusercontent.com/44917347/166919871-dad6b2bc-0c90-4449-af7a-b0c1b031a895.png)

Reply:
@mohitb35 Please verify the zoom level 
The left window seems to e more zoomed-in than right

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r867877084

Reply:
Fixed in #1467